fe/src/components/Statistics/StatisticsChart/StatisticsChart.re

```diff
@@ -0,0 +1,50 @@
+open EloTypes;
+open BsRecharts;
+
+let component = ReasonReact.statelessComponent(__FILE__);
+
+let renderUserLine = user => {
+  let colorOpts = StatisticsColors.randomColorOptions(~seed=user.code, ());
+  let color = StatisticsColors.randomColor(colorOpts);
+
+  <Line
+    key={user.userNid |> string_of_int}
+    name={user.code}
+    dataKey={user.code}
+    stroke=color
+    dot=false
+    connectNulls=true
+  />;
+};
+
+let renderChart = (users, checkedUsersNids, data) => {
+  let filterOutcheckedUsersNids = (checkedUsersNids, user) => {
+    checkedUsersNids |> List.exists(nid => nid === user.userNid);
+  };
+
+  let usersToshow =
+    if (List.length(checkedUsersNids) === 0) {
```

Comment:
Jedyne co to tylko niewiem po co to jest. Przez to gdy ktoś ręcznie odznaczy wszystkich to maluje mu się 5 linii, nie lepiej poprostu wtedy malować pusty wykres zgodnie z życzeniem usera ?

Reply:
@ktrzos 
Jeszcze nie ustosunkowałeś się do tego :P
Ale spoko, to nie jest istotne, merguje. Najwyżej później się zmieni